src/plasmapy/simulation/particle_tracker/termination_conditions.py

```diff
@@ -0,0 +1,222 @@
+"""
+Module containing termination conditions for the particle tracker.
+"""
+
+__all__ = [
+    "AbstractTerminationCondition",
+    "TimeElapsedTerminationCondition",
+    "NoParticlesOnGridsTerminationCondition",
+    "AllParticlesOffGridTerminationCondition",
+]
+
+from abc import ABC, abstractmethod
+
+import astropy.units as u
+import numpy as np
+import numpy.typing as npt
+
+
+class AbstractTerminationCondition(ABC):
+    """Abstract base class containing the necessary methods for a ParticleTracker termination condition."""
+
+    @property
+    def tracker(self):
+        """Return the |ParticleTracker| object for this termination condition."""
+        return self._particle_tracker
+
+    @tracker.setter
+    def tracker(self, particle_tracker) -> None:
+        self._particle_tracker = particle_tracker
+
+    @property
+    @abstractmethod
+    def require_synchronized_dt(self) -> bool:
+        """Return if this termination condition requires a synchronized time step."""
+        ...
+
+    @property
+    @abstractmethod
+    def progress_description(self) -> str:
+        """Return a small string describing the relevant quantity shown on the meter."""
+        ...
+
+    @property
+    @abstractmethod
+    def units_string(self) -> str:
+        """Return a string representation of the units of `total`."""
+        ...
+
+    @property
+    @abstractmethod
+    def is_finished(self) -> bool:
+        """Return `True` if the simulation has finished."""
```

Comment:
```suggestion
        """Return ``True`` if the simulation has finished."""
```

I guess maybe this needs double quotes?

Reply:
I think the issue is that it wasn't capitalized before